lib/src/postgrest_error.dart

```diff
@@ -25,4 +25,10 @@ class PostgrestError {
         'hint': hint,
         'code': code,
       };
+
+  @override
+  String toString() {
+    return 'PostgrestError{message: $message, '
+        'details: ${details?.toString()}, hint: $hint, code: $code}';
```

Comment:
IMO the code should be after message because details and hint are usually null

Reply:
@bdlukaa Reordered these variables! Let me know what you think!

Reply:
It's good!